### DISCUSSION TITLE
How does manually adjusting the mesh work?

### DISCUSSION THREAD

**Discussion by apr1967** [Jyers/Marlin#645] [General Discussion]
I looked and I cant find anything about it in the wiki.

**Comment by Jyers** [comment]
What config are you referring to? BL Touch, UBL, ManualMesh?

**Reply by apr1967** [reply]
From the leveling menu it says manually adjust mesh or something similar.  I would assume it wouldn't matter who I did for creating the mesh, its says manual adjustment.  Maybe I need to go look in Meyers wiki for it.

**Reply by Jyers** [reply]
It is just a way to modify individual mesh points after the fact, go to point, check offset, etc.
